app/seqexec-server-test-l64/src/templates/bash-template

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env bash
```

Comment:
There is no need to review this script. It is taken from sbt-native-packager and slightly modified to support running on a custom jre. Hopefully sbt-native-packager will take a PR with this change